bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/BallerinaErrors.java

```diff
@@ -72,76 +70,4 @@ static ErrorValue createNumericConversionError(Object inputValue, BType targetTy
                                      RuntimeErrors.INCOMPATIBLE_SIMPLE_TYPE_CONVERT_OPERATION,
                                      TypeChecker.getType(inputValue), inputValue, targetType));
     }
-
-    public static String getPrintableStackTrace(ErrorValue error) {
-        return getPrintableStackTrace(getErrorMessage(error), error.getStackTrace());
```

Comment:
Move this to ErrorValue class so we can override throwable printstacktrace method and use e.printstacktrace for ErrorValue as well.